.eslintrc

```diff
@@ -43,36 +50,39 @@ rules:
   prefer-spread: 2
   prefer-template: 2
 
-globals:
-  # Meteor globals
-  Meteor: false
-  DDP: false
-  Mongo: false
-  Session: false
-  Accounts: false
-  Template: false
-  Blaze: false
-  UI: false
-  Match: false
-  check: false
-  Tracker: false
-  Deps: false
-  ReactiveVar: false
-  EJSON: false
-  HTTP: false
-  Email: false
-  Assets: false
-  Handlebars: false
-  Package: false
-  App: false
-  Npm: false
-  Tinytest: false
-  Random: false
-  HTML: false
+  # eslint-plugin-meteor
+  ## Meteor API
+  meteor/globals: 2
+  meteor/core: 2
+  meteor/pubsub: 2
+  meteor/methods: 2
+  meteor/check: 2
+  meteor/connections: 2
+  meteor/collections: 2
+  meteor/session: [2, 'no-equal']
+
+  ## Best practices
+  meteor/no-session: 0
```

Comment:
There was a proposal to deprecate usage of `Session`. In case any application followed this proposal, using `Session` can be explicitly forbidden with this rule.

The rule is disabled by default, so this line is redundant. All other rules are set to error out.